src/Oro/Bundle/DistributionBundle/OroKernel.php

```diff
@@ -134,9 +134,13 @@ protected function collectBundles()
         $exclusions = array();
         foreach ($files as $file) {
             $import  = Yaml::parse($file);
-            $bundles = array_merge($bundles, $this->getBundlesMapping($import['bundles']));
-            if (!empty($import['exclusions'])) {
-                $exclusions = array_merge($exclusions, $this->getBundlesMapping($import['exclusions']));
+            if (null !== $import){
```

Comment:
I think that this check here is redundant because in lines 138 and 141 we have all the necessary checks.